### DISCUSSION TITLE
Dynamic class names in Sveltekit not working?

### DISCUSSION THREAD

**Discussion by MarcSallent** [saadeghi/daisyui#2660] [Q&A] [answered]
I do this with my own styles, but doesn't work with daisyui classes:

I create a button like this:

`<button class="btn btn-{status}">Send</button>`

Where status is dynamic, thus getting classes like btn-error, btn-success, etc.

This will not work unless I "hardcode" a button with btn-error, btn-success classes in the same page. I guess svelte doesn't import those classes unless it finds them in the code?

**Comment by MarcSallent** [comment]
I ended up doing:

```
<button class="btn 
    class:btn-success={status=="success"}
    class:btn-error={status=="error"}
    class:btn-loading={status=="loading"}
    disabled="{status=='loading'}">
...
 ```

**Comment by saadeghi** [comment, marked as answer]
Read this please: https://tailwindcss.com/docs/content-configuration#dynamic-class-names
